driver-legacy/src/main/com/mongodb/DB.java

```diff
@@ -83,7 +84,7 @@
         this.name = name;
         this.executor = executor;
         this.collectionCache = new ConcurrentHashMap<String, DBCollection>();
-        this.commandCodec = MongoClient.getCommandCodec();
+        this.commandCodec = new DBObjectCodec(getDefaultCodecRegistry());
```

Comment:
This was the only user of the MongoClient method, so just inlined it.

Reply:
Ack